pyartcd/pyartcd/util.py

```diff
@@ -267,3 +268,28 @@ async def is_build_permitted(version: str, data_path: str = constants.OCP_BUILD_
 
     # Fallback to default
     return True
+
+
+def get_release_controller_arch(releaseStreamName):
```

Comment:
Let's keep the `lowercase_with_underscore` style for function params

Reply:
Also I think we would have something in doozer/elliott doing this, maybe we import that here?